scss/_cards.scss

```diff
@@ -42,6 +44,19 @@
   }
 }
 
+.card-warning, .card-default {
+  .card-blockquote {
+    color: $card-blockquote-color;
+  }
+}
+
+.card-block {
+  > .card-block {
```

Comment:
Hmm not sure and searching for selector rules that depend on hierarchy is a problem.  I'll remove and deal with this is other ways.  Might be old legacy junk.